### PR TITLE
Fix: Translate App Strings to Chinese (Simplified zh_hans)

### LINKED ISSUE #2344: Translate App Strings to Chinese (Simplified zh_hans)
### User Story

As a Chinese language speaker, I want to be able to interact with Status in my native language.

### Description

*Type*: Feature

*Summary*: We need someone fluent (and eloquent) in Chinese to translate all the latest strings in Status from English. We would like to invite our community members to take on these tasks in order
a) To become a bit more acquainted with the Status repos and perhaps wander further into the codebase and set of issues we need help with, earning yourself Open Bounties all the way down this rabbit hole, and
b) To make sure that the product we deliver has been built by the same community that uses it, in a virtuous feedback loop of increasing understanding and value-generation.

#### Expected behavior
The Chinese translation ought to be as up-to-date as the English one.

#### Actual behavior
We have not yet updated it, preferring to leave this open as an opportunity for our community to get involved again.

### Solution

*Summary*: 

- The file [here](https://github.com/status-im/status-react/blob/develop/src/status_im/translations/zh_hans.cljs) needs to be updated to match the new English one [here](https://github.com/status-im/status-react/blob/develop/src/status_im/translations/en.cljs).
- How you translate is a matter of your preference. There were significant updates to the English file for the 0.9.12 release, with lots of strings both add and REMOVED. We need you to make sure that the files are exact mirrors of each other, except in different languages.
- It might be easier to simply take the new English file and translate from scratch rather than going through the current translated files line by line (though we leave this up to you).

### Additional Information

* Status version: 0.9.12+

There are bounties available for these translations. Please see the QR codes that will be added below after the DevCon 3 announcement.

### PR DESCRIPTION
**Pull request by kaichaosun**
fixes #2344

### CODE REVIEW

File: src/status_im/translations/zh_hans.cljs [hunk 1]
```diff
@@ -11,55 +11,57 @@
...
    :type-a-command                        "开始输入指令..."
    :error                                 "错误"
+   :unknown-status-go-error               "未知的status-go错误"
+   :node-unavailable                      "没有运行的以太坊节点"
+   :yes                                   "是"
+   :no                                    "否"
 
-   :camera-access-error                   "要授予所需的摄像机许可，请转到系统设置，并确定选择了“状态”>“摄像机”。"
-   :photos-access-error                   "要授予所需的照片许可，请转到系统设置，并确保选择“状态“>“照片“。"
+   :camera-access-error                   "要授予所需的摄像机许可，请转到系统设置，并确定选中了“状态”>“摄像机”。"
```
Comment (ninadou) [inline src/status_im/translations/zh_hans.cljs]: We don't translate Status this project name into Chinese, so we just call it "Status".
"要授予所需的照片许可，请转到系统设置，并确保选择"Status">"照片"。"

File: src/status_im/translations/zh_hans.cljs [hunk 2]
```diff
@@ -11,55 +11,57 @@
...
+   :photos-access-error                   "要授予所需的照片许可，请转到系统设置，并确保选中了“状态“>“照片“。"
```
Comment (ninadou) [inline src/status_im/translations/zh_hans.cljs]: "要授予所需的照片许可，请转到系统设置，并确保选中了"Status">"照片"。"

File: src/status_im/translations/zh_hans.cljs [hunk 3]
```diff
@@ -69,38 +71,32 @@
...
    :message                               "信息"
-   :username                              "用户名"
    :not-specified                         "未指定"
    :public-key                            "公钥"
    :phone-number                          "电话号码"
-   :email                                 "电子邮件"
    :update-status                         "更新你的状态..."
    :add-a-status                          "添加状态..."
-   :status-prompt                         "建立一个状态以帮助人们了解您提供的内容。 你也可以使用#hashtags。"
+   :status-prompt                         "设置你的状态。使用 #hastags 可以帮助其他人了解您，并且谈论您正在想些什么。"
```
Comment (ninadou) [inline src/status_im/translations/zh_hans.cljs]: "设置你的 Status。使用 #hastags 可以帮助其他人了解您，并且谈论您正在想些什么。"

**Comment by ninadou** [comment]
Hi, I think most of the translations are fine. But what we call Status is just "Status" not the translation term. So I think strings including "Status" should be fixed.

**Comment by kaichaosun** [comment]
Thanks for the review @ninadou .

**Review by andytudhope — APPROVED** [review]

**Pull request merged**